sdks/python/apache_beam/ml/transforms/tft.py

```diff
@@ -196,14 +195,17 @@ def apply_transform(
       data = self._split_string_with_delimiter(
           data, self.split_string_by_delimiter)
 
+    vocab_filename = self._vocab_filename
+    if vocab_filename:
```

Comment:
since we are moving away from default, what would be the name if `self._vocab_filename` is None? 

(Don't know if this was already answered somewhere else)

Reply:
oh tft takes care of it. Got it, thanks!

Reply:
I think if None is provided, a filename will be chosen from the current scope and counter (if there are multiple similar files) is attached to it by tft

For example, for the value None - I see `['vocab_compute_and_apply_vocabulary_vocabulary', 'vocab_compute_and_apply_vocabulary_1_vocabulary']`

Reply:
okay, thanks!